doc/classes/Vector2.xml

```diff
@@ -111,6 +111,7 @@
 			<argument index="0" name="with" type="Vector2" />
 			<description>
 				Returns the cross product of this vector and [code]with[/code].
```

Comment:
Maybe:
```suggestion
				Returns the 2D analog of the cross product for this vector and [code]with[/code].
```
That's how it's described on https://mathworld.wolfram.com/CrossProduct.html